{"Launchpad":"https://bugs.launchpad.net/bugs/1478971","Id":"1478971","Gerrit":"https://review.openstack.org/207529","IdGerrit":"207529","IdCommit":"c0f6a6e9e9b8713d12a02b8441c7616c24871526","Files":["cinder/service.py"],"Lines_Inseted":5,"Lines_Deleted":0,"CommitParent":"9652106985a301b6eedf08819f5155da01058457","Classification":"Bug","KeywordsTitle":"exception, kills thread","Description":"failed","KeywordsCommit":"should not kill, noticed, fixed, not handled, error","Comments":""}